pyartcd/pyartcd/slack.py

```
@@ -31,6 +32,33 @@ def __init__(
+        """Resolve an email address to a Slack user ID using the users.lookupByEmail API.
+        Returns None if the lookup fails (e.g. user not found, API error).
+        """
+        try:
+            # Use the synchronous WebClient for this one-off lookup at init time
+            from slack_sdk import WebClient
+            sync_client = WebClient(token=self.token)
+            response = sync_client.users_lookupByEmail(email=email)
+            user_id = response["user"]["id"]
+            _LOGGER.info("Resolved email %s to Slack user ID %s", email, user_id)
+            return user_id
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)